src/modules/Dashboard/components/BuildOptionsSection/BuildOptionsSection.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import styles from './BuildOptionsSection.scss';
```

Comment:
shouldn't be uppercase. Make first letter lowercased, because we only call components with uppercase, but this is not a component